### PR TITLE
Update gif in Write tests using Postbot

### PR DESCRIPTION
**Pull request by akinard-postman**
This updates the gif in the _Write tests using Postbot_ section to match the current UI. In reviewing the written workflow it appears that it remains the same. I scaled the image up as the current Postbot changes resulted in a larger screen presence (and a larger gif).

### CODE REVIEW

File: src/pages/docs/writing-scripts/test-scripts.md [hunk 1]
```diff
@@ -186,7 +186,7 @@ To write a test with Postbot, do the following:
 1. In the upper right corner of the test editor, select the Postbot icon <img alt="Postbot icon" src="https://assets.postman.com/postman-docs/v10/icon-postbot-v10-16.jpg#icon" width="18px">.
 1. Enter your query as a simple text sentence, or select one of the suggested queries to tell Postbot what you need.
 
-      <img alt="Postbot writing a test" src="https://assets.postman.com/postman-docs/v10/postbot-add-test-v10-16.gif" width="300px" />
+      <img alt="Postbot writing a test" src="https://assets.postman.com/postman-docs/v10/postbot-add-test-v10-21-11-2.gif" width="800px" />
```
Comment (mark-anthony-postman) [inline src/pages/docs/writing-scripts/test-scripts.md:189]:
```suggestion
      <img alt="Postbot writing a test" src="https://assets.postman.com/postman-docs/v10/postbot-add-test-v10-21-11-2.gif"" />
```

Comment (mark-anthony-postman) [reply src/pages/docs/writing-scripts/test-scripts.md:189]: For images this wide, I recommend not specifying a width in the HTML. That way it users can click the image and open it in a larger view if needed.

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/writing-scripts/test-scripts.md (hunk 1 above)
Comment (akinard-postman) [reply src/pages/docs/writing-scripts/test-scripts.md]: Thanks for the tip; I've removed the sizing.

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

**Pull request merged**